scss/_base_code.scss

```diff
@@ -16,10 +16,15 @@
     hyphens: none;
     line-height: 1.5;
     margin-bottom: 0;
+    margin-top: 0;
     tab-size: 4;
     white-space: pre-wrap;
     word-spacing: normal;
     word-wrap: break-word;
+
+    & + & {
+      @include default-vertical-spacing;
```

Comment:
It was previously blocked on #1138 but that's now landed and I've rebased so it's now all good.